test/gurka/test_osrm_serializer.cc

```diff
@@ -53,11 +53,12 @@ TEST(Standalone, HeadingNumberTurnLeft) {
                                  {{"/shape_format", "geojson"}});
   //
   auto json = gurka::convert_to_json(result, Options::Format::Options_Format_osrm);
-  auto heading_number{1};
-  for (int i = 0; i < json["routes"][0]["legs"][0]["steps"].GetArray().Size(); ++i)
-    ASSERT_EQ(heading_number, json["routes"][0]["legs"][0]["steps"][0]["intersections"][0]["bearings"]
-                                  .GetArray()
-                                  .Size());
+  std::vector<int> expected_headings{1, 2, 1};
```

Comment:
Based on the other tests, I also modified this cycle to test all the expected headings.
Now, variable `i` in the cycle is been used